### ISSUE TITLE
`require("fzf-lua").get_last_query()` may be broken

### ISSUE THREAD

**Issue by xiaochuanyu** [ibhagwan/fzf-lua#942]
### Info

- Operating System: OSX
- Shell: zsh
- Terminal: iTerm
- `nvim --version`:  v0.9.4
- `fzf --version`: 0.32.1

- [x] The issue is reproducible with `mini.sh`

### Description 
I do:
```
:lua require("fzf-lua").live_grep()
```
I type "anything" then press esc to close fzf window.
```
lua vim.print(require("fzf-lua").get_last_query())
```
^ this prints `nil` but expected `anything`.

Not reproducible after reverting to commit before f32b5c736439d8431b5e16c893a1169ff8a1b3a2.

**Comment by xiaochuanyu** [comment]
Ah ok yea still resume works, its just `get_last_query` returning `nil`.
My use-case was basically just resuming grep with different `rg_opts` using `get_last_query` but looks like I can just do something this now:
```
require("fzf-lua").live_grep({resume=true, rg_opts=my_custom_rg_opts})
```

**Comment by ibhagwan** [comment]
> Ah ok yea still resume works, its just `get_last_query` returning `nil`. My use-case was basically just resuming grep with different `rg_opts` using `get_last_query` but looks like I can just do something like this now:
> 
> ```lua
> require("fzf-lua").live_grep({resume=true, rg_opts=my_custom_rg_opts})
> ```

Correct, that would be the most perferred way, in any event https://github.com/ibhagwan/fzf-lua/commit/077221e31f04aa3e54fcdd69df91e390cc3fb6eb fixes the issue in the OP as well.
